app/templates/editions/octane.hbs

```diff
@@ -1,53 +1,118 @@
 {{title "Editions"}}
 {{title "Octane"}}
+
 <section class="container" aria-labelledby="octane-edition">
-  <h1 id="octane-edition">The Octane Edition of Ember</h1>
+  <h1>The Octane Edition of Ember</h1>
+  <p class="intro">
+    It has arrived!
+    As of version 3.15 of Ember, when you generate a new app, you get everything you need to build your app using Octane.
+    You can read more about features and upgrade strategies in the
+    <a href="https://blog.emberjs.com/2019/12/20/octane-is-here.html">the official Ember blog post</a>.
+  </p>
   <p class="mb-3">
-    The <strong>preview</strong> of Ember Octane has begun. This means that everything here is experimental and subject to change. Keep that in mind while you peruse this information!
+    Ember Octane describes a set of new features that, when taken together, represent a foundational improvement to the way you use Ember.
+    It has modern, streamlined components and state management that make it fun to build web applications.
+    With seamless interoperability for existing apps, teams can migrate at their own pace, while developers building new apps start out with the best that Ember has to offer.
   </p>
-  <ul class="unstyled-list grid lg:grid-4">
-    <EsCard class="lg:col-3" card-link>
+  <ul class="unstyled-list grid lg:grid-5">
+    <EsCard card-link>
+      <h3>
+        <a href="https://blog.emberjs.com/2019/12/20/octane-is-here.html">Read the announcement</a>
+      </h3>
+    </EsCard>
+    <EsCard card-link>
```

Comment:
what is the "card-link" part mean?

Reply:
This is documented in https://deploy-preview-145--ember-styleguide.netlify.com/components/card/ and is a way to enable the "stretch link" functionality described in https://inclusive-components.design/cards/